src/forms/output/BenefitsLines.js

```diff
@@ -131,7 +131,8 @@ class BenefitsLinesComp extends Component {
       lines.push(line);
     }
 
-    // Get 'Unexpected template string expression' warning otherwise
+    // Label for split tooltip 'labels'/'label headers' that appear
+    // at the bottom. Really long.
     // @todo Abstract commonalities between graphs
     const labelHeaderFormatStart = `<span style="font-size: 10px">${getText(snippets.i_beforeMoney)}`,
```

Comment:
:( inline styles

(I know this is pre-existing, just makes me sad)

Reply:
Yeah, sorry. I meant to hunt down that selector.

Reply:
If you have the time, feel free to tackle this one.

Reply:
@todo